src/compiler/parser.ts

```diff
@@ -2902,6 +2901,7 @@ namespace ts {
                 case SyntaxKind.UnknownKeyword:
                 case SyntaxKind.StringKeyword:
                 case SyntaxKind.NumberKeyword:
+                case SyntaxKind.BigIntKeyword:

```

Comment:
In parseJsonText you need to handle numeric literal checks (to ignore the big ints)

Reply:
I separated `SyntaxKind.BigIntLiteral` from `SyntaxKind.NumericLiteral`, so I believe that is no longer necessary